src/components/EventsTable.js

```diff
@@ -17,6 +17,7 @@ import moment from "moment";
 import { CSVLink } from "react-csv";
 
 import { FETCH_EVENTS_QUERY } from "../util/graphql";
+import { FETCH_EVENTS_REVERSED_QUERY } from "../util/graphql";
```

Comment:
this is giving an error, saying there is no FETCH_EVENTS_REVERSED_QUERY from ../util/graphql
furthermore it would be best practice to edit the existing FETCH_EVENTS_QUERY to take an additional parameter that determines the order that the events are returned; as opposed to making an identical method that accomplishes the same thing,